app/models/allocation.rb

```diff
@@ -34,7 +30,11 @@ class Allocation < ApplicationRecord
   def days_until_finish
     return unless end_at
 
-    end_at > Date.current ? (end_at - Date.current).to_i : I18n.t('finished')
+    if ongoing
+      (end_at - Date.current).to_i
```

Comment:
This may result in negative numbers. Is this an expected behavior?

Reply:
yep